zebra-chain/src/transparent.rs

```diff
@@ -156,3 +182,14 @@ pub struct Output {
     /// The lock script defines the conditions under which this output can be spent.
     pub lock_script: Script,
 }
+
+impl Output {
+    /// Get the value balance of this output.
+    ///
+    /// Needed to calculate the transparent value balance.
+    pub fn value_balance(&self) -> Amount<NegativeAllowed> {
```

Comment:
Documentation/Method Name: This is a value or amount.

(The input and output value balances have opposite signs, but in this code, the values have the same sign. That's ok, we just need to document it.)

Can we change the method name as well?
`value` and `amount` are both accurate names.

```suggestion
    /// Get the value contained in this output.
    /// This amount is subtracted from the transaction value pool by this output.
    pub fn value(&self) -> Amount<NegativeAllowed> {
```